app/utils/misc.ts

```diff
@@ -101,3 +102,8 @@ export function useDoubleCheck() {
 
 	return { doubleCheck, getButtonProps }
 }
+
+type ClassNameValue = string[] | string | null | undefined | 0 | false;
+export function consolidate(...inputs: ClassNameValue[]) {
+	return twMerge(inputs)
```

Comment:
If this is all this does, then I think it makes more sense to just use twMerge directly instead of having our own wrapper function.